frontend/app/model/MembershipSummary.scala

```diff
@@ -0,0 +1,17 @@
+package model
+
+import org.joda.time.DateTime
+
+case class MembershipSummary(startDate: DateTime,
+                             firstPaymentEndDate: DateTime,
+                             amountPaidToday: Float,
+                             planAmount: Float,
+                             nextPaymentPrice: Float,
+                             nextPaymentDate: DateTime,
+                             renewalDate: DateTime,
+                             initialFreePeriodOffer: Boolean) {
+
+
+  val annual = startDate.plusYears(1) == renewalDate
```

Comment:
Feels a bit over complicated, we use this kind of logic in a few places. Both dates are coming from Zuora which I assume is able to account for the leap date so the existing logic should work?